.golangci.yml

```diff
@@ -3,9 +3,41 @@ run:
   tests: false
 
 linters:
-  enable-all: true
```

Comment:
The `enable-all` option is deprecated, since if you use it, a bunch of new linters you never heard of and probably don't want (like the very idiosyncratic `wsl`) will go into effect whenever you upgrade golangci-lint. So I've added a specific subset that I think we want.